### PR TITLE
build: build generates then builds

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

**Comment by wtrocki** [comment]
Requires rebase. Sorry

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -21,4 +21,13 @@ jobs:
         node-version: '14'
     - run: yarn install
     - run: yarn generate
+    - name: Build kafka-management-sdk
```
Comment (wtrocki) [inline .github/workflows/build.yaml:24]: This should be eather:

- Script
- Single command based on parent (preferred)


Update contributing when done

Comment (secondsun) [reply .github/workflows/build.yaml:24]: We don't have a parent any more, the sdk are three separate maven projects. I'll script it up.

**Comment by secondsun** [comment]
rebased and updated

**Review by wtrocki — APPROVED** [review]

**Pull request merged**